site/app/templates/grading/simple/Display.twig

```diff
@@ -94,19 +120,13 @@
                 <th style="width:20px">#</th>
                 <th style="width:70px" scope="col">Section</th>
                 <th style="width:90px; text-align: left" scope="col">
-                    <a href="{{ grading_url }}?view={{ view_all ? 'all' : '' }}&sort=id" aria-label="sort by ID">
-                        <span class="tooltiptext" title="sort by User ID">User ID </span><i class="fas fa-sort"></i>
-                    </a>
+                    <span class="tooltiptext" title="sort by User ID">User ID </span>
```

Comment:
Given that we've removed the sort on these headers, we should probably also remove the `tooltiptext` as well since it's no longer valid.

This carries for the changes below here as well.